loguru.hpp

```diff
@@ -102,6 +102,11 @@ Website: www.ilikebigbits.com
 #include <sal.h>	// Needed for _In_z_ etc annotations
 #endif
 
+#if !defined(_MSC_VER)
+#include <syslog.h>
+#endif
```

Comment:
Done.

Reply:
Had to put it back.
The LOG_USER macro is used as a default value to `add_syslog()` function.

Reply:
Did you still want this changed?

Reply:
Yes please

One of the goals of Loguru is to make the `loguru.hpp` header as light weight as possible to keep compile times low for all users. To that end, `loguru.hpp` does not `#include` any other headers (except on Windows, because reasons), and I'd like to keep it that way.

One way to accomplish this here would be to have `int facility = -1` as the default, and in `loguru.cpp` detect that as the `LOG_USER` case.